substrate/client/network/src/request_responses.rs

```diff
@@ -893,13 +821,13 @@ impl NetworkBehaviour for RequestResponsesBehaviour {
 									}
 
 									if response_tx
-										.send(Err(RequestFailure::Network(error.clone().into())))
+										.send(Err(RequestFailure::Network(error.clone())))
 										.is_err()
 									{
 										log::debug!(
 											target: "sub-libp2p",
 											"Request with id {:?} failed. At the same time local \
-											 node is no longer interested in the result.",
+											node is no longer interested in the result.",
```

Comment:
`rustfmt` change?

Reply:
No, this is my change to avoid double whitespace in logs

Reply:
Reverted